### ISSUE TITLE
Don't require external CSS to show attribution

### ISSUE THREAD

**Issue by sansumbrella** [mapbox/mapbox-gl-js#9353]
## Motivation

Correctly attributing Mapbox on a map should be the easy default when using the gl-js library.

Currently, application developers must add a line of code in order to add attribution to their maps. This is easily forgotten in an HTML page, and a bit tricky to figure out how to do when using a bundler like Webpack. This makes it harder to correctly attribute Mapbox than it is to leave off attribution.

## Design Alternatives

Require application code to _remove_ attribution.

- Render the attribution within the WebGL context
- Inject the CSS into the page at runtime as part of the gl-js library.

## Current state
With external CSS loaded (attribution appears correctly):
![pink map of the US with yellow heatmap and attribution across the bottom](https://user-images.githubusercontent.com/81553/75467918-61258900-595a-11ea-8b9d-2af8b39a422f.png)

Without external CSS loaded (no attribution):
![pink map of the US with yellow heatmap](https://user-images.githubusercontent.com/81553/75467992-7e5a5780-595a-11ea-93b7-e92cfa90b90a.png)

HTML pages must source the correct (matching) version of CSS to add attribution, e.g.:
`<link href="https://api.mapbox.com/mapbox-gl-js/v1.8.0/mapbox-gl.css" rel="stylesheet" />`

Bundled applications can source the CSS from their node_modules folder and importing it close to where they use Mapbox, e.g.:
`import 'mapbox-gl/dist/mapbox-gl.css';`

**Comment by mourner** [comment]
Note that adding `mapbox-gl.css` alongside `mapbox-gl.js` is required for correct operation of the map in general — otherwise many features won't work, not just attribution: all the controls (buttons, compass, scale control, search etc.), cursors, interactions like shift-drag-zoom, HTML markers and popups, etc. 

Currently, if you don't include CSS, there will be a warning about it in the developer console. We don't want to support CSS-less GL JS usage at all, so moving attribution support to JS is not a good option. Would you rather suggest disabling the map completely in this case?

**Comment by andrewharvey** [comment]
I agree with @mourner here, the css is required, and don't think we should try and support a CSS-less GL JS. I think the current console warning is enough.
